frontend/apps/reader/modules/readerlink.lua

```diff
@@ -306,6 +307,21 @@ function ReaderLink:addToMainMenu(menu_items)
             })
         end,
     }
+    menu_items.go_to_next_location = {
+        text = _("Go Forward to next location"),
+        enabled_func = function() return self.forward_location_stack and #self.forward_location_stack > 0 end,
+        callback = function() self:onGoForwardLink() end,
+        hold_callback = function(touchmenu_instance)
+            UIManager:show(ConfirmBox:new{
+                text = _("Clear Forward location history?"),
```

Comment:
Missed this, but weird capitalization. But the better solution is to just remove this completely imho. What's the purpose?

Reply:
`Go Forward to next location` has the same problem